code/src/utils/Emoji.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Emoji = props => (
+    <span
+        className="emoji"
+        role="img"
+        aria-label={props.label ? props.label : ""}
+        aria-hidden={props.label ? "false" : "true"}
+    >
+        {props.symbol}
+    </span>
+)
```

Comment:
Really nice idea to put the span tags and aria labels within the component, I haven't seen that before!